features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/UserFromConfigEventScenario.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UserFromConfigEventScenario.swift
+//  iOSTestApp
+//
+//  Created by Jamie Lynch on 26/05/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import Foundation
+import Bugsnag
+
+/**
+ * Sends an event  to Bugsnag which contains a user set from Configuration
+ */
+internal class UserFromConfigEventScenario: Scenario {
+
+    override func startBugsnag() {
+        self.config.autoTrackSessions = false;
+        self.config.setUser("abc", withEmail: "fake@gmail.com", andName: "Fay K")
+        super.startBugsnag()
+    }
+
+    override func run() {
+        let user = Bugsnag.user()
+        Bugsnag.addMetadata(user.id, key: "id", section: "clientUserValue")
```

Comment:
It's unclear to me what this is doing and could maybe do with a comment?
My best guess is that it is taking the value from `Bugsnag.user()` and putting it in metadata so you can assert that the correct values were set by passing in config. Is that right?

Reply:
There is a default user information which has been temporarily removed while working around an issue with the buildkite pipeline - a ticket exists to enable this again once that issue has been resolved.

This line is achieving the following test requirement, I'll add a comment to clarify this for future readers:

>Setting the user in the Configuration and verifying its presence using getUser on the Client, Event, and Session (Suggest acquiring the user and setting metadata to the same value e.g. receivedClientUser)